CODE-OF-CONDUCT.md

```diff
@@ -27,6 +27,17 @@ In keeping with this commitment, we offer the following guidelines:
      attend meetings.
    * Work performed within this group, either finalized or in draft, is to be used in accordance with the group [Mission and Charter](https://github.com/cncf/tag-security/blob/main/governance/charter.md), the open source license, and to be used for the equal benefit of all members of the community.  Further information on use of work may be found in [Security Assessments: Outcome](https://github.com/cncf/tag-security/tree/main/assessments#outcome)
 
+
+# Incident handling and escalation
+
+If an incident that occurs that you is not in good fair, please follow the guidelines on handling and escalating the issue to TAG leadership:
+
+- If you see content that clearly does not meet the official Code of Conduct, please make it known to a Co-Chair/TL and the creator of the content. (For more details refer to the CNCF Code of Conduct).
```

Comment:
please use new chair email list:  https://github.com/cncf/tag-security/issues/655 -- if a single chair or TL is afk, we don't want someone to get upset that they reported something and nothing was done.  Reporting must not include a single point of failure.

Reply:
Yup agree on no single point on failure, this PR depends on #655. (i wish github had a way to enforce this)

Reply:
fair -> faith?

Reply:
```suggestion
- If you see content that clearly does not meet the official Code of Conduct, please send an e-mail to the Co-Chair/TL mailing list (cncf-tag-security-leads@lists.cncf.io) and the creator of the content. (For more details refer to the CNCF Code of Conduct).
```